.gitignore

```diff
@@ -27,5 +27,4 @@ node_modules
 # Users Environment Variables
 .lock-wscript
 
-docs/
```

Comment:
According to this documentation, it should be in docs/: https://github.com/canjs/canjs/pull/3011/files